src/routes/routes.ts

```diff
@@ -44,6 +44,7 @@ export const ROOT_ROUTE = '/'
 export const WELCOME_ROUTE = '/welcome'
 export const OPEN_SAFE_ROUTE = '/open'
 export const LOAD_SAFE_ROUTE = generatePath(LOAD_SPECIFIC_SAFE_ROUTE) // By providing no slug, we get '/load'
+export const SAFE_APP_LANDPAGE_ROUTE = '/share/safe-app'
```

Comment:
I would rename this to `SAFE_APP_LANDING_PAGE_ROUTE`.

Reply:
done!